code/game/objects/structures/catwalk.dm

```diff
@@ -128,6 +128,12 @@
 							break
 					update_icon()
 
+/obj/structure/catwalk/hoist_act(turf/dest)
+	for(var/A in loc)
+		var/atom/movable/AM = A
+		AM.forceMove(dest)
```

Comment:
probably needs anchored / simulated checks so you're not lifting lighting overlays